tus_storagehandler/app.py

```diff
@@ -0,0 +1,72 @@
+"""API server entry point."""
+
+import logging
+import os
+from pathlib import Path
+from typing import Optional
+
+from connexion import FlaskApp
+from foca import Foca
+
+logger = logging.getLogger(__name__)
+
+
+def init_app() -> FlaskApp:
+    """Initialize and return the FOCA app.
+
+    This function initializes the FOCA app by loading the configuration
+    from the environment variable `TUS_FOCA_CONFIG_PATH` if set, or from
+    the default path if not. It raises a `FileNotFoundError` if the
+    configuration file is not found.
+
+    Returns:
+        A Connexion application instance.
+
+    Raises:
+        FileNotFoundError: If the configuration file is not found.
+    """
+    # Determine the configuration path
+    if config_path_env := os.getenv("TUS_FOCA_CONFIG_PATH"):
+        print(config_path_env)
+        config_path = Path(config_path_env).resolve()
+    else:
+        config_path = (
+            Path(__file__).parents[1] / "deployment" / "config.yaml"
+        ).resolve()
+
+    # Check if the configuration file exists
+    if not config_path.exists():
+        raise FileNotFoundError(f"Config file not found at: {config_path}")
+
+    foca = Foca(
+        config_file=config_path,
+    )
+    return foca.create_app()
+
+
+def main(env: Optional[str] = None) -> None:
+    """Run FOCA application.
+
+    Args:
+        env (str, optional): The environment in which to run the application.
+                             Defaults to 'prod' if not specified.
```

Comment:
I think it's safer to set a development/testing environment as default. Generally, when setting defaults, always pick the safer option. That way you make sure that people _actively_ have to choose the less safe option.

Also, please list all the acceptable values and consider defining (and validating against) a corresponding enum.